Cargo.toml

```
@@ -27,6 +27,11 @@ no-panic = { version = "0.1", optional = true }
 num_cpus = "1.8"
 rand = "0.8"
 rand_xorshift = "0.3"
+criterion = "0.4"
 
 [package.metadata.docs.rs]
 targets = ["x86_64-unknown-linux-gnu"]
+
+[[bench]]
+name = "bench"
+harness = false
```

I would add a new line at the end of the file, as in other files
```suggestion
harness = false

```